### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zoykhan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,31 +4,29 @@
 -->
 # Overview
 
-OPS stands for operational simplicity which is achieved by "on-the-box" automation in python, on IOS-XR routers.
+This repository contain samples of python scripts for "on-the-box" automation on IOS-XR routers. For other type of scripts and their support details please reach out to Cisco Systems Inc or refer to the IOS-XR release manuals.
```
Comment (songfeijun) [inline README.md:7]: "other type of script" , perhaps "scripts in other languages"?

File: README.md [hunk 2]
```diff
@@ -4,31 +4,29 @@
 -->
 # Overview
 
-OPS stands for operational simplicity which is achieved by "on-the-box" automation in python, on IOS-XR routers.
+This repository contain samples of python scripts for "on-the-box" automation on IOS-XR routers. For other type of scripts and their support details please reach out to Cisco Systems Inc or refer to the IOS-XR release manuals. 
 
-This feature is supported on IOS-XR releases 7.3.2 onwards.
+This feature was introduced in 7.3.2 IOS-XR release with limited support and additional functional support is added in subsequent releases as outlined in Cisco's release documentation.
```
Comment (songfeijun) [inline README.md:9]: excellent.

File: README.md [hunk 3]
```diff
@@ -4,31 +4,29 @@
...
 
 ## 3. EEM Script ##
 
-The difference between an eem script and an exec script is that the eem script is triggered via a predefined set of events whereas the exec script is triggered by a CLI(user). In 7.4.1 and 7.3.2 IOS-XR releases, the only event that is supported is syslog. 7.5.1 onwards, we also have timer, track and telemetry events support along with logical correlation of events, rate-limit, occurrence and period. 
-
-Every eem script has a maxrun (default is 20 seconds and can be overridden via config). If script doesn't terminate in "maxrun" time, script is killed.
+An EEM (Embedded Event Manager) script is triggered via a predefined set of events. Events supported are syslog, timer, traceback and telemetry events along with logical correlation of events, rate-limit, occurrence and period. Every eem script has a maxrun (default is 20 seconds and can be overridden via config). If script doesn't terminate in "maxrun" time, the script is killed.
```
Comment (songfeijun) [inline README.md:25]: "if script doesn't" 
"if the script doesn't"

File: README.md [hunk 4]
```diff
@@ -4,31 +4,29 @@
...
-
-A process script is quite different as it will try to run forever as part of its design, and if it exits due to abnormal reason (it crashes itself, some other processes kill it, etc.), an external entity will try to restart the process. In another word, there is an external entity to do the “life cycle management” of the script.
+A process script will try to run forever as part of its design, and if it exits due to abnormal reason (it crashes itself, some other processes kill it, etc.), an external entity will try to restart the process. In another word, there is an external entity to do the “life cycle management” of the script. It is different compared to exec/eem/config scripts that are transient in nature and start running due to an external trigger (a XR-CLI, an event or a commit action), they run for a short period of time and then cease running. If these scripts cease running as part of normal code flow, or due to a mishap (it crashes itself, or some other processes kills it), no entity will attempt to run it again without next external trigger.
```
Comment (songfeijun) [inline README.md:29]: "abnormal reason"
"abnormal reasons"

Comment (songfeijun) [inline README.md:29]: "an external entity will try"
"XR will try"

Comment (songfeijun) [inline README.md:29]: "there is an external entity to do the"
"XR is doing" or "XR will do"

Comment (songfeijun) [inline README.md:29]: "if these scripts cases"
"if an exec/eem/config script ceases"

**Review by songfeijun — APPROVED** [review]

**Comment by zoykhan** [comment]
Thank you for the review. I have made the changes

**Pull request merged**